specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-10-26-preview/examples/Operations_List.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Don't need this example, if pervious example of Operations_List can work.

Reply:
Removed as suggested.